style.css

```diff
@@ -0,0 +1,23 @@
+#title {
```

Comment:
I think you should try to use classes as a selector to follow the semantics of CSS.

Reply:
Good job on using inline comments here. The comment is also polite and you gave a reason for your suggestion.:+1:
[OPTIONAL] Try to elaborate more on the reason with more sentences. You can reference any lesson for the student to read more in your comment.